src/scales/scale.time.js

```diff
@@ -635,11 +661,11 @@ module.exports = function(Chart) {
 			if (helpers.isObject(value)) {
 				label = me.getRightValue(value);
 			}
-			if (timeOpts.tooltipFormat) {
-				label = momentify(label, timeOpts).format(timeOpts.tooltipFormat);
+			if (typeof label === 'string') {
```

Comment:
That doesn't work because if `tooltipFormat` is given, we still want to format the value :\

Reply:
good catch. I've updated again. hopefully it's good now